cdflow_commands/plugins/aws_lambda.py

```diff
@@ -174,11 +174,13 @@ def create(self):
 
     def _zip_up_component(self):
         logger.info('Zipping up ./{} folder'.format(self._component_name))
+        top_level = os.getcwd()
         with ZipFile(self._component_name + '.zip', 'w') as zipped_folder:
             for dirname, subdirs, files in os.walk(self._component_name):
-                zipped_folder.write(dirname)
+                os.chdir(dirname)
```

Comment:
I didn't realise you could use a context manager.. I prefer that too so I'll change it